agent/p/agent/github/verification/task/definition.go

```diff
@@ -0,0 +1,30 @@
+package task
+
+import (
+	"bufio"
+	"bytes"
+)
+
+const Type string = "gh-verification"
+
+type Definition struct {
+	Handle  string
+	Address string
+}
+
+func (d Definition) MarshalJSON() ([]byte, error) {
+	buf := new(bytes.Buffer)
+	w := bufio.NewWriter(buf)
+
+	w.WriteString(
+		`{"handle":"` + d.Handle +
+			`","address":"` + d.Address + `"}`,
+	)
+
+	w.Flush()
+	return buf.Bytes(), nil
+}
+
```

Comment:
I'm working on a PoC to allow passing data types using maketx call.

Reply:
Something like this?
```go
--- contract.gno ---
package withtypes

type A struct {
  Value int
}

func DoSomething(a A) { ... }
```
`gnokey maketx call -func DoSomething -args "{\"Value\": 1}" -argtypes withtypes.A`